src/pybind/cephfs/cephfs.pyx

```diff
@@ -156,18 +156,15 @@ cdef extern from "cephfs/libcephfs.h" nogil:
 
 
 class Error(Exception):
-    pass
-
-
-class OSError(Error):
```

Comment:
what's the rationale for this change? commit message didn't explain

Reply:
I see, you want this for commit "cephfs-shell: catch libcephfs.Error only in onecmd()". The better thing to do IMO is catch `libcephfs.OSError` instead.

Reply:
> what's the rationale for this change? commit message didn't explain

Wanted `perror(e.strerror)` work for both exceptions. IMO `class Error` accepting different arguments did not make sense. I confirmed about this from Varsha since she changed/defined what args `class Error` takes.

Reply:
> I see, you want this for commit "cephfs-shell: catch libcephfs.Error only in onecmd()". The better thing to do IMO is catch `libcephfs.OSError` instead.

Didn't want to miss out on [`LibCephFSStateError`](https://github.com/ceph/ceph/blob/master/src/pybind/cephfs/cephfs.pyx#L204)

Reply:
> Didn't want to miss catching [`LibCephFSStateError`](https://github.com/ceph/ceph/blob/master/src/pybind/cephfs/cephfs.pyx#L204).

Okay. But I still don't think making the `libcephfs.Error` class always have a strerror/errno field is correct. I think the better approach is to use the __str__ method to generate a message. I'll point elsewhere in this review.